src/GoogleAnalyticsEventForwarder.js

```diff
@@ -313,16 +313,17 @@
 
                     sendEcommerceEvent(event.EventCategory, outputDimensionsAndMetrics, customFlags);
                 }
-                else if (event.ProductAction.ProductActionType == mParticle.ProductActionType.Checkout) {
+                else if (event.ProductAction.ProductActionType == mParticle.ProductActionType.Checkout || event.ProductAction.ProductActionType == mParticle.ProductActionType.CheckoutOption) {
+                    var type = event.ProductAction.ProductActionType;
```

Comment:
it looks like this variable is unused, not sure if you meant to use it later when you generate the command string or if it can just be removed